test/e2e/lifecycle/main_test.go

```diff
@@ -39,6 +40,11 @@ var _ = BeforeSuite(func() {
 	By("Adding custom resource scheme to framework")
 	err := framework.AddToFrameworkScheme(apis.AddToScheme, &cnaov1.NetworkAddonsConfigList{})
 	Expect(err).ToNot(HaveOccurred())
+
+	err = framework.AddToFrameworkScheme(monitoringv1.AddToScheme, &monitoringv1.ServiceMonitorList{})
+	Expect(err).ToNot(HaveOccurred())
+	err = framework.AddToFrameworkScheme(monitoringv1.AddToScheme, &monitoringv1.PrometheusRuleList{})
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Use one line per check with `Exec(framework.AddToFrameworkSchema...).To(Succeed())`

Reply:
done